### ISSUE TITLE
tf.math.angle not working in tflite concrete function

### ISSUE THREAD

**Issue by zeynepgulhanuslu** [tensorflow/tensorflow#54147]
Hi,
When i try to use tf.math.angle function inside concrete function and save as tflite model it gives an error like below:

Some ops are not supported by the native TFLite runtime, you can enable TF kernels fallback using TF Select

tf version 2.7 

Can you help me with this?

Thanks

**Comment by tilakrayal** [comment]
@zeynepgulhanuslu ,
In order to expedite the trouble-shooting process, could you please provide the complete code and dataset to reproduce the issue reported here.

**Comment by zeynepgulhanuslu** [comment]
Complete code is like below.
`
import numpy as np
import tensorflow as tf

print(tf.__version__)


class MyFeatExtractor(tf.Module):
    def __init__(self,
                 frame_length=512,  # in samples, assumes 25ms of 16khz audio
                 frame_shift=128):
        super().__init__()
        self.frame_length = frame_length
        self.frame_shift = frame_shift
        self.fft_length = int(2 ** np.ceil(np.log2(self.frame_length)))
        self.window_type = "povey"
        if self.window_type == "povey":
            self.window = tf.reshape(
                tf.pow(tf.signal.hann_window(self.frame_length, periodic=False),
                       0.85), (1, 1, -1))
        else:
            raise NotImplementedError  # TODO: Implement others

    @tf.function
    def __call__(self, pcm):
        return self.calculate(pcm)

    def calculate(self, pcm):
        # Gather frames
        framed_signals = tf.signal.frame(pcm, self.frame_length,
                                         self.frame_shift, pad_end=False)
        # Remove DC offset
        means = tf.expand_dims(
            tf.reduce_mean(framed_signals, axis=-1, keepdims=False, name=None),
            -1)
        framed_signals -= means

        # Apply window
        framed_signals *= self.window
        # Apply FFT
        framed_signals = tf.pad(framed_signals,
                                [[0, 0], [0, 0],
                                 [0, self.fft_length - self.frame_length]])
        fft_result = tf.signal.rfft(framed_signals, [self.fft_length])
        fft_real_part = tf.math.real(fft_result)
        fft_imag_part = tf.math.imag(fft_result)
        fft_abs = tf.math.abs(fft_result)
        fft_phase = tf.math.angle(fft_result)
        return fft_abs, fft_phase


if __name__ == '__main__':
    feat_type = "custom_feat"
    feat_ex = MyFeatExtractor()

    concrete_func = feat_ex.__call__.get_concrete_function(
        pcm=tf.TensorSpec(shape=[1, None], dtype=tf.float32))

    # Convert the model
    converter = tf.lite.TFLiteConverter.from_concrete_functions([concrete_func])
    #converter.experimental_new_converter = True
    converter.allow_custom_ops = True

    converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS, tf.lite.OpsSet.SELECT_TF_OPS]

    converter.experimental_enable_mlir_converter = True
    tflite_model = converter.convert()

    # Save the model.
    with open(f'tf_{feat_type}_calculator.tflite', 'wb') as f:
        f.write(tflite_model)
`
I just want to create a model that generates rfft angle and abs value. Not using any dataset.

Thanks

**Comment by tilakrayal** [comment]
@zeynepgulhanuslu ,
I was able to execute the mentioned code without any error.Please find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/334e1187362654deec81d8f49cf1c8f0/untitled202.ipynb).Please confirm or provide the error log which you are facing the issue.

**Comment by zeynepgulhanuslu** [comment]
Thank you for your reply. The error like below:
```
TFLite interpreter needs to link Flex delegate in order to run the model since it contains the following Select TFop(s):
Flex ops: FlexAngle
Details:
	tf.Angle(tensor<?x?x257xcomplex<f32>>) -> (tensor<?x?x257xf32>) : {device = ""}```

**Comment by zeynepgulhanuslu** [comment]
Hi,
